### PR TITLE
pylint fixes

### PR DESCRIPTION
**Pull request by marksmayo**
Simplifying some statements / code blocks, fixing deprecated logging calls, and a couple of others.

### CODE REVIEW

File: pylenium/element.py [hunk 1]
```diff
@@ -1385,7 +1349,7 @@ def apply_style(s):
             self._py.webdriver.execute_script("arguments[0].setAttribute('style', arguments[1]);", self.webelement, s)
 
         original_style = self.webelement.get_attribute("style")
-        apply_style("border: {0}px solid {1};".format(border, color))
+        apply_style(f"border: {0}px solid {1};".format(border, color))
```
Comment (ElSnoMan) [inline pylenium/element.py:1352]: Either use an f-string or use .format()

**Review by ElSnoMan — CHANGES_REQUESTED** [review]
Thanks for cleaning things up and submitting a Pull Request! I have one small request, but otherwise, it all looks great! I just merged a branch to `main`, which you'll probably want to merge to this branch, but hopefully there are no conflicts

File: pylenium/element.py (hunk 1 above)
Comment (marksmayo) [reply pylenium/element.py:1352]: fixed in latest commit post merge

**Comment by marksmayo** [comment]
@ElSnoMan looks like your merge broke one test? test_element_actions.py, seems unable to find a file?

**Review by ElSnoMan — APPROVED** [review]

**Comment by ElSnoMan** [comment]
> @ElSnoMan looks like your merge broke one test? test_element_actions.py, seems unable to find a file?

It's been resolved, so no worries. Your PR has been merged 🎉

**Pull request merged**